index.js

```diff
@@ -0,0 +1,47 @@
+// set up express
+var express = require('express'); // taking express from the package 
+var app = express();
+var port = 3000;
+
+// create a route for the homepage
+
+app.get('/', function(request, response){
+    response.send("welcome to the terms dictionary!!");
+})
+
+
+// make ur app listen 
+app.listen(port, function(){
+    console.log(`App is listening on port ${port}`)
+}) 
+
+//import terms defintions 
+
+var definitions = require('./definitions')
+
+// create a route for server 
+
+app.get('/definitions', function(request, response){
```

Comment:
you are missing the :term params `app.get('/definitions/:term', function(request, response){`